mm2src/mm2.rs

```diff
@@ -76,6 +80,41 @@ impl LpMainParams {
     }
 }
 
+#[cfg(test)]
+pub fn password_policy(_password: &str) -> (bool, &str) { (true, "") }
+
+#[cfg(not(test))]
+pub fn password_policy(password: &str) -> (bool, &str) {
+    lazy_static! {
+        static ref REGEX_NUMBER: Regex = Regex::new(".*[0-9].*").unwrap();
+        static ref REGEX_LOWERCASE: Regex = Regex::new(".*[a-z].*").unwrap();
+        static ref REGEX_UPPERCASE: Regex = Regex::new(".*[A-Z].*").unwrap();
+        static ref REGEX_SPECIFIC_CHARS: Regex = Regex::new(".*[*.!@#$%^&(){}:;'<>,.?/~`_+-=|].*").unwrap();
+    }
+    if password.to_lowercase().contains("password") {
+        return (false, "contains the word password");
+    }
+    if password.len() < 8 || password.len() > 32 {
+        return (false, "password length should be between 8 and 32");
+    }
+    if !REGEX_NUMBER.is_match(password) {
+        return (false, "password should contains at least 1 digit");
+    }
+    if !REGEX_LOWERCASE.is_match(password) {
+        return (false, "password should contains at least 1 lowercase character");
+    }
+    if !REGEX_UPPERCASE.is_match(password) {
+        return (false, "password should contains at least 1 uppercase character");
+    }
+    if !REGEX_SPECIFIC_CHARS.is_match(password) {
+        return (false, "password should contains at least 1 specific character");
```

Comment:
I was able to set the `pass123worD` password and run mm2. It seems that there's something wrong with Regex. Please recheck it and cover this fn with unit tests.

Reply:
`should contains`  -> `should contain` (remove `s`)
`specific` -> `special`
`password can't contains 3 times the same number of characters` -> `Password can't contain the same character 3 times in a row`